packages/babel-generator/src/generators/flow.js

```diff
@@ -262,6 +262,19 @@ function andSeparator() {
   this.space();
 }
 
+export function InterfaceTypeAnnotation(node: Object) {
+  this.word("interface");
+  this.space();
```

Comment:
I think that this can be removed, since it is already called at lines 269 and 274.